### PR TITLE
Update Windows Terminal Steps

### LINKED ISSUE #2: Update Windows Terminal steps
The steps for setting up Windows terminal needs extending to include:

- [x] Michael's set up [video tutorial](https://youtu.be/wPmwgMm0plk)
- [x] Installing VS Code via the windows subsystem
- [x] Installing node via the windows subsystem
- [x] Setup home path alias
- [x] Setup and use SSH with GitHub
- [x] Setup default GitBranch on git init
- [x] Persistent SSH keys

### PR DESCRIPTION
**Pull request by haelmj**
- [x] Added Michael's setup video tutorial
- [x] Added steps to set up VS Code on WSL
- [x] Added steps to install Node via WSL
- [x]  Steps for setting up home path alias
- [x] Setup GCM core for PAT persistence (wsl)
- [x] Setup SSH Keys
- [x] Setup default Git Branch for git init

When merged, fixes #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,14 +3,18 @@
 This guide is to help you setup the following required software during the Black Codher bootcamp.
 
 1. [Windows Terminal](#windows-terminal) **THIS IS FOR WINDOWS USERS ONLY**
+1. [Setup Home Path Alias](#setup-home-path-alias-in-windows-terminal) **THIS IS FOR WINDOWS USERS ONLY**
 1. [Git](#git)
+   - [Setup SSH Keys for GitLab](#setup-ssh-keys-for-gitlab)
```
Comment (tanyapowell) [inline README.md:8]: GitLab?

File: README.md [hunk 2]
```diff
@@ -98,19 +116,76 @@ The instructions below mention **Git Bash** and **Windows command prompt**, you
...
+
+
 ## Node
 
 You will need to install Node not only for the NodeJS unit, but also for React. By installing Node you can use its a [package manager](https://en.wikipedia.org/wiki/Package_manager) (Node Package Manager or `npm`) to install other software that you will use in your projects later in the course.
 
 ### How to install node
 
-[Click on this link](https://nodejs.org/en/download/) and choose the installer appropriate for your computer (windows or mac). Follow the instructions the installer gives.
+[Click on this link](https://nodejs.org/en/download/) and choose the installer appropriate for your computer ( mac). Follow the instructions the installer gives.
```
Comment (tanyapowell) [inline README.md:177]: Extra gap before `mac`

File: README.md [hunk 3]
```diff
@@ -98,19 +116,76 @@ The instructions below mention **Git Bash** and **Windows command prompt**, you
...
+- Then run `./script.sh`
+- You should receive the following output or something similar:
+![Script Output](scriptOutput.png)
+
+#### Version 2.28 and above
+- Run `git config --global init.defaultBranch main`
+- Close the Terminal.
+
+
+
```
Comment (tanyapowell) [inline README.md:170]: No need for these extra lines

File: README.md [hunk 4]
```diff
@@ -98,19 +116,76 @@ The instructions below mention **Git Bash** and **Windows command prompt**, you
...
+### Configure Default Branch for Git Init
+
+- Verify your version of Git by running `git --version`
+If your version number is below 2.28, then proceed with the steps below. Otherwise, skip to the steps under Version 2.28.
+
+#### Below Version 2.28
+- Open the Terminal
+- Download the configure script by running: `wget https://raw.githubusercontent.com/blackcodherbootcamp/setup-guide/main/script.sh`
+- Then run `./script.sh`
```
Comment (tanyapowell) [inline README.md:161]: How do they run this script?

**Review by tanyapowell — COMMENTED** [review]
Starting to think having mac and windows instruction on one page is going to cause confusion. Going to separate them, so there's a page for windows and one for mac

**Comment by tanyapowell** [comment]
@haelmj I've opened a PR - #4 - splitting up the guide. So one page for windows and one for mac

File: README.md (hunk 1 above)
Comment (haelmj) [reply README.md:8]: My bad! I've been using GitLab for work for the past couple of months. It's taken over my life. I'll correct it now.

File: README.md (hunk 4 above)
Comment (haelmj) [reply README.md:161]: They just have to put the command on line 161 and hit enter.

Comment (haelmj) [reply README.md:161]: Line 160 gets them to download the file and then line 161 executes it

File: README.md [hunk 5]
```diff
@@ -98,19 +116,74 @@ The instructions below mention **Git Bash** and **Windows command prompt**, you
...
+- Open up Windows Terminal
+- Enter this command and run: `git config --global credential.helper "/mnt/c/Program\ Files/Git/mingw64/libexec/git-core/git-credential-manager-core.exe"`
+
 ### Mac installation
 
 [Follow the instructions](https://github.com/git-guides/install-git#install-git-on-mac).
 
 ---
 
+## Setup SSH Keys for GitHub
```
[the remark this answers is not in the document]
Comment (haelmj) [reply README.md:134]: Yes this works for both devices

**Pull request merged**